lms/templates/courseware/course_about.html

```diff
@@ -18,7 +18,10 @@
 </%block>
 
 <%block name="js_extra">
-
+  ## CourseTalk widget js script
+  % if show_coursetalk_widget:
+      <script src="//d3q6qq2zt8nhwv.cloudfront.net/s/js/widgets/coursetalk-read-reviews.js"></script>
```

Comment:
I guess creating an NPM module will be the best solution, we will create and publish an npm or bower modules in further. However it will take a lot of time to do it right now.
What do you think if we deploy our script with source map? As for my opinion js with source map more easy to read in compare with not minified JS.